codegens/nodejs-request/test/unit/snippet.test.js

```diff
@@ -263,6 +263,46 @@ describe('nodejs-request convert function', function () {
       });
     });
 
+    it('should return snippet with proper semicolon placed where required', function () {
+      // testing for the below snippet
+      /*
+       var request = require('request');
+       var fs = require('fs');
+       var options = {
+         'method': 'GET',
+         'url': 'https://postman-echo.com/headers',
+         'headers': {
+           'my-sample-header': 'Lorem ipsum dolor sit amet',
+           'not-disabled-header': 'ENABLED'
+         }
+       };
+       request(options, function (error, response) {
+         if (error) throw new Error(error);
+         console.log(response.body);
+       }); */
+      request = new sdk.Request(mainCollection.item[0].request);
+      options = {};
+      convert(request, options, function (error, snippet) {
+        if (error) {
+          expect.fail(null, null, error);
+        }
+        console.log(snippet);
+        expect(snippet).to.be.a('string');
+        var snippetArray = snippet.split('\n');
+        snippetArray.forEach(function (line, index) {
```

Comment:
hm...tests seem a little roundabout. Can we do an exact string comparison check instead, and comments on the part of the string we're actually testing for?